tools/events/decode.go

```diff
@@ -0,0 +1,63 @@
+package event_tools
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	filTypes "github.com/filecoin-project/lotus/chain/types"
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime"
+	"github.com/ipld/go-ipld-prime/codec/dagcbor"
+	"github.com/ipld/go-ipld-prime/datamodel"
+)
+
+const (
+	cidEntryValues = "unsealed-cid,piece-cid"
```

Comment:
Have we checked on events docs if there are other cid entries available? Maybe some regex looking for the work "cid"? I would check it to be sure. Otherwise we will miss other cases.

Reply:
Yes, we have. But yes the regex is a good idea